New-EmptyLabs.ps1

```diff
@@ -28,6 +28,19 @@ $config | ForEach-Object {
       Write-Host "Creating Resource Group '$($_.ResourceGroupName)' ..." -ForegroundColor Green
       New-AzResourceGroup -Name $_.ResourceGroupName -Location $_.LabRegion | Out-Null
     }
+
+    # If specified create any/all the resource groups where VMs should be created
+    # The SilentlyContinue bit is to suppress the error that otherwise this generates.
+
+    if ($_.VmCreationResourceGroupName) {
+        
+        $existingVmCreationRg = Get-AzResourceGroup -Name $_.VmCreationResourceGroupName -Location $_.LabRegion -ErrorAction SilentlyContinue
+
+        if(-not $existingVmCreationRg) {
+          Write-Host "Creating Resource Group '$($_.ResourceGroupName)' ..." -ForegroundColor Green
```

Comment:
We should say "Creating resource group XYZ for lab ABC" just so we know which lab the resource group is used for in the log